app/containers/popup.tsx

```diff
@@ -164,45 +160,45 @@ export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
               </svg>
               <span>INTERCEPTOR</span>
             </a>
-            <button type="button" onClick={this.handleClick} className={buttonClass}>
-              {this.props.enabled ? "Stop Listening" : "Start Listening"}
+            <button
+              title={enabled ? "Stop Listening to Requests" : "Start Listening to Requests"}
```

Comment:
use `props.enabled` to make it consistent across the component. No need for `enabled` as separate const.